plugins/modules/monitoring/pagerduty_user.py

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Zainab Alsaffar <Zainab.Alsaffar@mail.rit.edu>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: pagerduty_user
+short_description: Manage a user account on PagerDuty
+description:
+    - This module manages the creation/removal of a user account on PagerDuty.
+version_added: '1.2.0'
```

Comment:
Quick note: 1.2.0 was released yesterday, so the next version will be 1.3.0 :)
```suggestion
version_added: '1.3.0'
```